sample-dcrwallet.conf

```diff
@@ -23,6 +23,9 @@
 ; flag.
 ; enablevoting=0
 
+; Account to autobuy tickets from.
+; purchaseaccount=
```

Comment:
Actually this is an autobuyer config therefore should be moved to `ticketbuyer.purchaseaccount`.

@jrick would it make sense to make this change in a separate PR ?

Reply:
no, that will break existing configurations

Reply:
yep, thought about that, then we leave it for the right moment